app/DoctrineMigrations/Version20171105202000.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Add origin_url column
+ */
+class Version20171105202000 extends AbstractMigration implements ContainerAwareInterface
+{
+    /**
+     * @var ContainerInterface
+     */
+    private $container;
+
+    public function setContainer(ContainerInterface $container = null)
+    {
+        $this->container = $container;
+    }
+
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        $entryTable = $schema->getTable($this->getTable('entry'));
+        
+        $this->skipIf($entryTable->hasColumn('origin_url'), 'It seems that you already played this migration.');
+        
+        $entryTable->addColumn('origin_url', 'text', [
+            'notnull' => true,
```

Comment:
Regarding test, I'm not sure about the not null. People can choose to let the origin url empty.

Reply:
Sure but null != empty :)

Reply:
Then you need to update some fixtures

Reply:
@j0k3r I don't want to see you here. Please disconnect yourself from the Internet 💋